templates/admin/maps_point_selector.djt

```diff
@@ -3,7 +3,7 @@
 {% block extrahead %}
 {{ block.super }}
 <script type="text/javascript" src="//www.ucf.edu/js/jquery.js"></script>
-<script src="//maps.google.com/maps/api/js?sensor=false&libraries=drawing " type="text/javascript" ></script>
+<script src="//maps.google.com/maps/api/js?key={{ google_api_key }}&sensor=false&libraries=drawing " type="text/javascript" ></script>
```

Comment:
Should the space at the end of this src="" be removed? Looks like it was there previously as well